src/resources/candlepin-docker-build.sh

```diff
@@ -10,5 +10,6 @@ docker --version
 docker login -u unused -p "$DOCKER_API_TOKEN" docker-registry.engineering.redhat.com
 ./docker/build-images -p -c
 
+sudo setenforce 0
```

Comment:
How complicated would it be to fix SELinux issues properly?

Reply:
I initially started trying to solve the selinux issues properly when I was writing the gating-test docker images build stuff months ago, but eventually came to the conclusion it would be simpler to do this. But I wouldn't be worried about it, because this is a temporary change. Currently there is an ongoing effort by @Lorquas and @Ceiu to refactor this whole thing: https://github.com/candlepin/candlepin/pull/2531
So turning off selinux here is a temporary solution, since all this will change anyway.

Reply:
OK, sounds good.